cloudshell-orch-core/cloudshell/workflow/orchestration/apps_configuration.py

```diff
@@ -49,16 +55,22 @@ def apply_apps_configurations(self, apps):
         apps_configuration = []
 
         for app in apps:
-            if len(app.app_request.appConfiguration) > 0:
-                apps_configuration.append(AppConfiguration(app.deployed_app.Name,
-                                                           app.app_request.appConfiguration))
+            if len(app.app_request.scripts) > 0:
+                scripts_configuration = []
+                for script in app.app_request.scripts.values():
+                    scripts_configuration.append(ConfigurationManagementData(
+                                                   ScriptAlias=script.scriptAlias,
+                                                   ConfigParams=script.scriptConfiguration
+                                               ))
+
+                apps_configuration.append(AppConfigurationData(app.deployed_app.Name, scripts_configuration))
                 self.sandbox.logger.debug(
                     "App '{0}' was added to appConfiguration using app request information".format(
                         app.deployed_app.Name))
 
 
             else:
-                apps_configuration.append(AppConfiguration(app.deployed_app.Name, None))
+                apps_configuration.append(AppConfigurationData(app.deployed_app.Name, None))
```

Comment:
even though the original code used None - consider using [] instead of None, it's safer in my opinion